### ISSUE TITLE
issues with custom tableviewcell

### ISSUE THREAD

**Issue by abenmaad** [KiranJasvanee/OnlyPictures#5]
Hi,
i used onlypicture in a custom uitableviewcell and when i scroll images disapear
there is a solution to clean images in prepareForReuse method
Thanks

**Comment by KiranJasvanee** [comment]
Hi,
I’ve used OnlyPictues in many screens with custom table view cell. I’ve not found issue related to image disappearing. But still I will check it, thanks for responding.

Sent from Yahoo Mail for iPhone


On Friday, February 9, 2018, 7:57 PM, abenmaad <notifications@github.com> wrote:


Hi,
i used onlypicture in a custom uitableviewcell and when i scroll images disapear
there is a solution to clean images in prepareForReuse method
Thanks

**Comment by abenmaad** [comment]
for your information i download image from url

**Comment by KiranJasvanee** [comment]
Yes. I’ve checked it. And it’s working fine with image downloading from url too. Make sure you are caching too. 


Sent from Yahoo Mail for iPhone


On Friday, February 9, 2018, 9:01 PM, abenmaad <notifications@github.com> wrote:


for your information i download image from url

**Comment by abenmaad** [comment]
i use alamofire to save image in cach but when i scroll image disapear or cell get another images

**Comment by abenmaad** [comment]
here attached a little example of the issue
[test.zip](https://github.com/KiranJasvanee/OnlyPictures/files/1747421/test.zip)
